src/components/singleBolgPage/sections/BlogInfo.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "../index.css";
+
+export const BlogInfo = () => {
+  return (
+    <div>
+      <img
```

Comment:
this PR has a lot of stuff thats not responsive donation stuff. can you use git to clean it up so it only contains the files i need to review?